content/en/docs/advanced/edgemesh.md

```diff
@@ -9,44 +9,9 @@ title: Edge connection with EdgeMesh
 toc: true
 type: docs
 ---
-In case network issue between cloud and edge side, we integrate EdgeMesh to support DNS visit at any time.
+In case of network issues between edge nodes, we integrate EdgeMesh to support the service discovery mechanism.
 
-Currently we only support HTTP1.x, more protocols like HTTPS and gRPC coming later.
-
-EdgeMesh is enabled as default.
-
-## Limitation
-
-* Ensure network interface "docker0" exists, which means that EdgeMesh only works for Docker CRI.
-
-## Environment Check
-
-Before run examples, please check environment first.
-
-### DNS Order
-
-Modify `/etc/nsswitch.conf`, make sure `dns` is first order, like below:
-
-```
-$ grep hosts /etc/nsswitch.conf
-hosts:          dns file mdns4_minimal [NOTFOUND=return]
-```
-
-### IP Forward Setting
-
-Enable ip forward:
-
-```
-$ sudo echo "net.ipv4.ip_forward = 1" >> /etc/sysctl.conf
-$ sudo sysctl -p
-```
-
-Then check it:
-
-```
-$ sudo sysctl -p | grep ip_forward
-net.ipv4.ip_forward = 1
-```
+Check more details at [edgemesh website](https://edgemesh.netlify.app) to deploy edgemesh application.
```

Comment:
```suggestion
Check more details at [edgemesh website](https://edgemesh.netlify.app) to use EdgeMesh.
```

Reply:
And all the content below can be deleted I think? users can ge them from  https://edgemesh.netlify.app